package.json

```diff
@@ -137,6 +137,11 @@
                     "type": "boolean",
                     "default": true,
                     "description": "Enable snippet completions"
+                },
+                "kite.enableOptionalCompletionsTriggers": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Enabling this will cause Kite to trigger completions after a space, ( and [. Note that this may cause completions from other providers from not showing up."
```

Comment:
"Note that this may cause completions from other providers from not showing up" -> 
"Note that this may cause completions from other providers to not show up"